demo/scripts/demo.gd

```diff
@@ -3,10 +3,31 @@ extends Node
 func _ready():
 	# Array2D
 	var array_2d = Array2D.new()
-	array_2d.resize(3, 4)
-	array_2d.set_cell(2, 3, "Array2D test")
-	print(array_2d.get_cell(2, 3))
-	
+	array_2d.resize(3, 3)
+	var count = 1
+	var data = array_2d.get_data()
+	for n in range(len(data)):
+		for m in range(len(data[0])):
+			array_2d.set_cell(n, m, count)
+			count += 1
+	print("Original")
+	print(array_2d.to_str())
+	print()
+	array_2d.transpose()
+	print("Transposed")
+	print(array_2d.to_str())
+	print()
+
+	var new_array_2d = array_2d.duplicate()
+	new_array_2d.set_cell(1,1, 300)
+	print("Original")
+	print(array_2d.to_str())
+	print()
+	print("Duplicated")
+	print(new_array_2d.to_str())
+	print()
```

Comment:
Did you mean to leave all of these print statements?

Reply:
Welp, nope. I that's just me debugging the transpose and the duplicate function which wasn't working properly.